src/xcode/ENA/ENA/Source/Developer Menu/Features/DMNotifications/DMNotificationsViewModel.swift

```diff
@@ -107,6 +111,23 @@ final class DMLocalNotificationsViewModel {
 
 		addNotification(request: request)
 	}
+	private func scheduleNotificationForInvalid(
+		id: String
+	) {
+		let content = UNMutableNotificationContent()
+		content.title = AppStrings.LocalNotifications.certificateGenericTitle
+		content.body = AppStrings.LocalNotifications.certificateGenericBody
+		content.sound = .default
+
+		let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 2.0, repeats: false)
```

Comment:
Does the tech-specs says to trigger it after two seconds?

Reply:
No, its in the dev menu...